app/controllers/protocols_controller.rb

```diff
@@ -901,10 +901,13 @@ def edit_description_modal
 
   # pio_stp_x means protocols io step (id of component) parser
   def pio_stp_1(iterating_key) # protocols io description parser
-    append = '<br>' + sanitize_input(iterating_key) + '<br>' if iterating_key.present?
-    append = t('protocols.protocols_io_import.comp_append.missing_desc') if iterating_key.blank?
-    return append if append.present?
-    ''
+    br = '<br>'
+    append = br + sanitize_input(iterating_key) + br if iterating_key.present?
+    if iterating_key.blank?
+      append = t('protocols.protocols_io_import.comp_append.missing_desc')
+      return append
```

Comment:
I think you don't need this `return append`. Or maybe you should change next line: instead od `append` use `''`.